botfront/cypress/integration/project_settings/project.settings.info.spec.js

```diff
@@ -22,7 +22,7 @@ describe('Project Settings', function() {
             cy.visit(`/project/${this.bf_project_id}/settings`);
             cy.get('.project-name input').click();
             cy.get('.project-name input').type('33');
-            cy.get('.save-project-info-button').click();
+            cy.contains('Save Changes').click();
```

Comment:
prefer data-cy if possible. We should only use "contains" for dropdown because it's driven by data. Otherwise if we change the text it will break the test

Reply:
Will come up with data-cy